game-core/src/main/java/games/strategy/triplea/ai/pro/util/ProMatches.java

```diff
@@ -345,49 +343,36 @@
 
   public static Predicate<Unit> unitCanBeMovedAndIsOwnedAir(final PlayerID player, final boolean isCombatMove) {
     return u -> {
-      if (isCombatMove && Matches.unitCanNotMoveDuringCombatMove().test(u)) {
-        return false;
-      }
-      return unitCanBeMovedAndIsOwned(player).and(Matches.unitIsAir()).test(u);
+      return (!isCombatMove || !Matches.unitCanNotMoveDuringCombatMove().test(u)) && unitCanBeMovedAndIsOwned(player)
+          .and(Matches.unitIsAir()).test(u);
     };
   }
 
   public static Predicate<Unit> unitCanBeMovedAndIsOwnedLand(final PlayerID player, final boolean isCombatMove) {
     return u -> {
-      if (isCombatMove && Matches.unitCanNotMoveDuringCombatMove().test(u)) {
-        return false;
-      }
-      return unitCanBeMovedAndIsOwned(player)
-          .and(Matches.unitIsLand())
-          .and(Matches.unitIsBeingTransported().negate())
-          .test(u);
+      return (!isCombatMove || !Matches.unitCanNotMoveDuringCombatMove().test(u)) && unitCanBeMovedAndIsOwned(player)
```

Comment:
formatting, dot operators each on their own line pls.
Clarifying parens would be really good here too.